samcli/commands/build/command.py

```diff
@@ -92,6 +92,12 @@
     help="If your functions depend on packages that have natively compiled dependencies, use this flag "
     "to build your function inside an AWS Lambda-like Docker container",
 )
+@click.option(
+    "--parallel",
+    "-p",
+    is_flag=True,
+    help="Use this flag to run builds of each function/layer in parallel using asyncio",
```

Comment:
Is "using asyncio" relevant for users?

Reply:
Not really, it might be helpful for some customers if they want to know what is under the hood. I will check with Mark about  any change related to this

Reply:
I see, it's a UX nit but I think in that case it might add cognitive overhead to users, as those unfamiliar might ask themselves "asyncio? what is it? why do I need to know about this? surely if it was mentioned I need to know about it?" etc.

Reply:
Yeah I understand your point, will raise it during doc discussions